config/api_server_deployment.yml

```diff
@@ -118,9 +118,9 @@ spec:
           resources:
             requests:
               cpu: 100m
-              memory: 300Mi
+              memory: 250Mi
             limits:
-              cpu: 500m
+              cpu: 1000m
```

Comment:
Increased this since nginx is in the critical path for API server requests. It's a concurrent web server so this could technically be increased further if we'd like.